src/bin/pgcopydb/catalog.c

```diff
@@ -3143,6 +3146,190 @@ catalog_iter_s_table_part_finish(SourceTablePartsIterator *iter)
 }
 
 
+/*
+ * catalog_iter_s_generated_column_next fetches the next GeneratedColumn entry
+ * in our catalogs.
+ */
+bool
+catalog_iter_s_generated_column_next(GeneratedColumnIterator *iter)
+{
```

Comment:
Nitpicking: the iterators are all declared in the same ordering (iter, init, next, fetch, finish, if I remember correctly). Could you follow the convention here?

Reply:
Done